### ISSUE TITLE
We need more people to help with this repository.

### ISSUE THREAD

**Issue by voyagermedical** [unshakenme/OSPAT#13]
Does anyone have any ideas on how we can attract more people to help with this? Perhaps post into relevant Reddit threads?

**Comment by voyagermedical** [comment]
There is this Reddit thread which I think we started by one of us...

https://www.reddit.com/r/opensource/comments/rsinc0/im_starting_an_open_source_psychedelicassisted/

Here are a few links to related Reddit articles:

https://www.reddit.com/r/therapists/comments/vd4hou/psychedelic_assisted_therapy/
https://www.reddit.com/r/PsychedelicTherapy/comments/ns5yh7/training_to_be_a_psychedelic_therapistcounselor/
https://www.reddit.com/r/PsychedelicTherapy/comments/kxv70f/becoming_a_psychedelic_assisted_therapist/

Perhaps, we could comment and get people more involved?

**Comment by unshakenme** [comment]
Absolutely! Please do invite people from around the internet here. The more the merrier :)

**Comment by unshakenme** [comment]
I had also started a community on Twitter to try and attract folks but haven't been able to keep up with it. If you want send me a DM on there and I can make you an admin https://twitter.com/i/communities/1506783971140505601

**Comment by voyagermedical** [comment]
Will do.

**Comment by CheekeyMonkey** [comment]
A good way to get more people here also would be for all of us to link our websites to the repository, this will increase the page rank and point more searches from Google to https://github.com/unshakenme/OSPAT. So if you have a website, or want to post about facebook think of what someone might search that might find this useful, and link the text like this...  i.e. [Free Psychedelic Therapy Course](https://github.com/unshakenme/OSPAT).

**Comment by unshakenme** [comment]
That's a great idea! I will post to my socials a few times over the coming weeks.

**Comment by CheekeyMonkey** [comment]
I have added an [open-source psychedelic-assisted therapy](https://en.wikipedia.org/wiki/Psychedelic_therapy#Open_Source_psychedelic_therapy) article to Wikipedia:

"Open Source psychedelic therapy
It is thought that there will eventually be 10,000 clinics for entactogen and psychedelic therapy established in the US[63]. To meet this demand, a huge workforce of therapists is needed. There are many courses which train healthcare professionals however, price has proven to be the main barrier to entry. As such in 2022 a new trend emerged, open source psychedelic therapy which offers the free training of clinicians in psychedelic therapy [Best Practice](https://en.wikipedia.org/wiki/Best_practice)[[64]](https://en.wikipedia.org/wiki/Psychedelic_therapy#cite_note-66)[[65]](https://en.wikipedia.org/wiki/Psychedelic_therapy#cite_note-67)."

If you have a Wikipedia account please add/edit it as this will make sure that it is not speedily deleted.

**Comment by unshakenme** [comment]
Wow that’s incredible. Good work!

**Comment by JDonMc** [comment]
@CheekeyMonkey curious that [64] has a website that also hosts a page on the Library of Babel, of which I own the discord community server for with about 660 members, and about 60 online at any given time.

https://burnzero.com/Library_of_Babel

But I also have:

https://github.com/JDonMc/PsychedelicTherapy

And that Burnzero article completely covers why I'm about to launch https://www.ClassA.education/
which also relates to Carhart Harris' latest research on the canalisation of over learning:
https://pubmed.ncbi.nlm.nih.gov/36584883/

**Comment by JDonMc** [comment]
But I also believe there should be an unlicense: https://unlicense.org/ equivalent for OSPAT, as well as a CC and copyleft.
ie. The ability to be a Psychedelic Therapist based purely on experience and research and learning, and effectiveness, and reputation, rather than an official accreditative license that depends on a 4 year degree. Imagine 4 years of daily Ayahuasca treatment!!!!

**Comment by JDonMc** [comment]
I know you guys are aiming to reach accreditors, or to become accreditors, but the underground movement has always been open-source-like with these things. So here Is a link to my repo that starts off with some writings by me, as well as also hosting a bunch of other well known manuals for this type of thing.

https://github.com/JDonMc/PsychedelicTherapy

**Comment by unshakenme** [comment]
Thanks for sharing your repo. There are a few resources on there that I don't think we've included here yet.

**Comment by CheekeyMonkey** [comment]
Yes would be great if you could see what we are missing, add and then submit via a pull request.

**Comment by JDonMc** [comment]
@CheekeyMonkey I actually disagree with the accreditation approach, as this has never been the way of the psychedelic crowd since the early days, and the underground today doesn't follow it. So I won't be submitting a pull request, as it is not clear to me what it even means to have a piece of paper that says you can help someone take psychedelics when you haven't even done them yourselves. I believe in an unlicense or a copyleft scenario to the accreditation. ie. no prior requirements, do as thou wilt, let the reputation of the product or service be the demand.

**Comment by unshakenme** [comment]
If you don't plan on submitting a pull-request to this repo then please refrain from spamming us with comments. This isn't a forum to discuss whether you agree or disagree with our overarching approach. Appreciate you sharing your repo and best of luck with it. If you change your mind in the future we'd be happy to review a pull-request.

**Comment by unshakenme** [comment]
@voyagermedical @CheekeyMonkey @a-n-g-i-e-r-i If you all are willing, please DM me your email address and I can setup an email string for us to talk about strategies for getting more contributors.

**Comment by a-n-g-i-e-r-i** [comment]
> @voyagermedical @CheekeyMonkey @a-n-g-i-e-r-i If you all are willing, please DM me your email address and I can setup an email string for us to talk about strategies for getting more contributors.

I don't think github has DMs but my email is "tristan.angieri at gmail dot com" and I am down to strategize with you. Just finished applying to grad school so will have more time for this project now

**Comment by JDonMc** [comment]
@unshakenme 99.9% of all feedback on all repos are not pull requests. That is not how this works. You have 1 repo, and have starred no repos, and are not writing any code. So I don't think you have any idea how this works. I came here because @voyagermedical invited me to contribute after he saw my repo in the same vein. So my contribution is a criticism. In Open Source terms it is a bug that either gets fixed or gets forked. You don't get to decide my behaviour as long as I'm not being abusive. You have yet to ADD anything of your own, all you've done is cite research, which is a key factor in the canalisation of overlearning: https://www.youtube.com/watch?v=4NtKdisg0GA&t=0s

you are not even a contributor yourself

for those who can't hear, you must feel.

**Comment by unshakenme** [comment]
Deleting comments unrelated to Issue topic per [rules](https://github.com/unshakenme/OSPAT/blob/main/CONTRIBUTING.md#rules)

**Comment by unshakenme** [comment]
@CheekeyMonkey do you have a preferred method for passing me your email address?

**Comment by RecoveringTherapist** [comment]
Hi there! I found this on Reddit. I’m unfamiliar with the platform but interested in contributing! Is this still active?

**Comment by CheekeyMonkey** [comment]
Hi, yes we are still active. Any help with the main repository would be welcome!

**Comment by unshakenme** [comment]
Hi @RecoveringTherapist yes we'd love any support you can offer. Take a look at the [Contributing page](https://github.com/unshakenme/OSPAT/blob/main/CONTRIBUTING.md) and let us know if you have any questions!
